ops/reth-cross-test/run.py

```
@@ -0,0 +1,136 @@
+from phases.sequencer_phase import PHASES, SequencerPhase
+from phases.stress_phase import StressPhase
+from phases.fault_phase import FaultPhase
+from phases.validator_phase import ValidatorPhase
+
+
+def prepare_images():
+    """Pull required Docker images."""
+    images = ["ghcr.io/morph-l2/morph-reth:latest"]
+    for img in images:
+        print(f"Pulling {img}...")
+        subprocess.run(["docker", "pull", img], check=False)
```

Ruff (0.15.7)

[error] `subprocess` call: check for execution of untrusted input

(S603)